_config.yml

```diff
@@ -58,75 +58,96 @@ flink_releases:
-      id: "1125-download-source"
-      url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.12.5/flink-1.12.5-src.tgz"
-      asc_url: "https://downloads.apache.org/flink/flink-1.12.5/flink-1.12.5-src.tgz.asc"
-      sha512_url: "https://downloads.apache.org/flink/flink-1.12.5/flink-1.12.5-src.tgz.sha512"
+      name: "Apache Flink 1.12.6"
+      id: "1126-download-source"
+      url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.12.6/flink-1.12.6-src.tgz"
+      asc_url: "https://downloads.apache.org/flink/flink-1.12.6/flink-1.12.6-src.tgz.asc"
+      sha512_url: "https://downloads.apache.org/flink/flink-1.12.6/flink-1.12.6-src.tgz.sha512"
     release_notes_url: "https://nightlies.apache.org/flink/flink-docs-release-1.12/release-notes/flink-1.12.html"
     optional_components:
       -
         name: "Avro SQL Format"
         category: "SQL Formats"
         scala_dependent: false
-        id: 1125-sql-format-avro
-        url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.12.5/flink-avro-1.12.5.jar
-        asc_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.12.5/flink-avro-1.12.5.jar.asc
-        sha_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.12.5/flink-avro-1.12.5.jar.sha1
+        id: 1126-sql-format-avro
+        url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.12.6/flink-avro-1.12.6.jar
+        asc_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.12.6/flink-avro-1.12.6.jar.asc
+        sha_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-avro/1.12.6/flink-avro-1.12.6.jar.sha1
+  -
+    version_short: "1.11"
+    binary_release:
+      name: "Apache Flink 1.11.5"
+      scala_211:
+        id: "1115-download_211"
+        url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.11.5/flink-1.11.5-bin-scala_2.11.tgz"
+        asc_url: "https://downloads.apache.org/flink/flink-1.11.5/flink-1.11.5-bin-scala_2.11.tgz.asc"
+        sha512_url: "https://downloads.apache.org/flink/flink-1.11.5/flink-1.11.5-bin-scala_2.11.tgz.sha512"
+      scala_212:
+        id: "1115-download_212"
+        url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.11.5/flink-1.11.5-bin-scala_2.12.tgz"
+        asc_url: "https://downloads.apache.org/flink/flink-1.11.5/flink-1.11.5-bin-scala_2.12.tgz.asc"
+        sha512_url: "https://downloads.apache.org/flink/flink-1.11.5/flink-1.11.5-bin-scala_2.12.tgz.sha512"
+    source_release:
+      name: "Apache Flink 1.11.5"
+      id: "1115-download-source"
+      url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.11.5/flink-1.11.5-src.tgz"
+      asc_url: "https://downloads.apache.org/flink/flink-1.11.5/flink-1.11.5-src.tgz.asc"
+      sha512_url: "https://downloads.apache.org/flink/flink-1.11.5/flink-1.11.5-src.tgz.sha512"
+    release_notes_url: "https://nightlies.apache.org/flink/flink-docs-release-1.11/release-notes/flink-1.11.html"
```

Comment:
Why flink-1.11.5 does not have `optional_components` such as `flink-avro`?

Reply:
Because I didn't want to dig into the history to figure out what exactly we listed for flink 1.11. In practice any optional component of the previous release will continue to work.